package-lock.json

```diff
@@ -4,47 +4,44 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "@electron/get": {
-      "version": "1.12.2",
-      "resolved": "https://registry.npmjs.org/@electron/get/-/get-1.12.2.tgz",
-      "integrity": "sha512-vAuHUbfvBQpYTJ5wB7uVIDq5c/Ry0fiTBMs7lnEYAo/qXXppIVcWdfBr57u6eRnKdVso7KSiH6p/LbQAG6Izrg==",
```

Comment:
Its making me a little concerned that stuff is still getting changed in the package and package-lock files. This is added later, but all of these types of packages shouldn't be changing every pr. Whatever is happening should be fixed soon so something important doesn't get deleted like this one. Although this one is added on later in the file, the package and package-lock shouldn't be changed at all by this point unless something is getting re-added. 

For the moment, if everything still works we don't need to change anything for this file in this pr.

@ARC-Summer-2020/arc-ui Starting on pull requests opened after this one, the package and package-lock files cannot be pushed unless we are adding a dependency.

I will link this comment in other prs that are currently open so people for sure see this.

Reply:
I should also note, that adding this to the .gitignore is not a valid solution because it will need to get changed sometimes. Just make sure it is not in the commits